src/drivers/px4io/px4io.cpp

```diff
@@ -3625,7 +3625,8 @@ px4io_main(int argc, char *argv[])
 			fn[1] =	"/fs/microsd/px4io1.bin";
 			fn[2] =	"/fs/microsd/px4io.bin";
 			fn[3] =	nullptr;
-#elif defined(CONFIG_ARCH_BOARD_PX4FMU_V2)
```

Comment:
Yes. I had a wip to do so but it got interrupted - this week is bad for me - but I will resume that work when I can.